chain/default_syncer_test.go

```diff
@@ -204,7 +208,7 @@ func loadSyncerFromRepo(t *testing.T, r repo.Repo, dstP *DefaultSyncerTestParams
 	chainStore := chain.NewDefaultStore(chainDS, calcGenBlk.Cid())
 
 	blockSource := th.NewTestFetcher()
-	syncer := chain.NewDefaultSyncer(cst, con, chainStore, blockSource) // note we use same cst for on and offline for tests
+	syncer := chain.NewDefaultSyncer(cst, con, chainStore, blockSource, 1) // note we use same cst for on and offline for tests
```

Comment:
putting a `1` here is not a good permanent solution (imo). As part of the "What's left" section of the PR, this will be addressed.

It is my opinion that a refactor of this file is in order -- its over 1000 lines long and was not easy to extend. However I believe said refactor is outside the scope of this PR.

Reply:
Messy code that's impeding some functionality, or testing thereof, that we wish to add is exactly the right signal to do a refactor. Please do such refactor in a preceding PR, though.

Reply:
Moving this validation check to a method on `con` is "the right thing" IMO.  Mocking out block times then becomes part of mocking out a consensus module.  Also I don't disagree about this file needing a refactor.

Reply:
Reiterating: this `1` should be moved into the constructor of the consensus validation module.